lap3dkernel/c++SIMD/VCL_AgnerFog_version1/vectorf512e.h

```diff
@@ -1346,6 +1346,11 @@ static inline Vec8d sqrt(Vec8d const & a) {
     return Vec8d(sqrt(a.get_low()), sqrt(a.get_high()));
 }
 
+// aprrox 1/sqrt(a) function, takes double type as function argument

```

Comment:
VCL version 1 or version 2 don't have simd approx rsqrt for double type argument, so I added to the local VCL lib.